rust/syntect-plugin/src/main.rs

```diff
@@ -329,19 +320,18 @@ impl<'a> Plugin for Syntect<'a> {
     fn config_changed(&mut self, _view: &mut View<Self::Cache>, _changes: &ConfigTable) {}
 
     fn update(&mut self, view: &mut View<Self::Cache>, delta: Option<&RopeDelta>,
-              _edit_type: String, _author: String) -> Option<u64> {
+              undo_group: Option<usize>, _edit_type: String, _author: String) {
```

Comment:
ditto earlier, all of syntect's edits will be part of the active undo group so we don't need to pass the argument around.